graph-builder/src/plugins/release_scrape_dockerv2/mod.rs

```diff
@@ -0,0 +1,146 @@
+//! This plugin scrapes a Docker V2 compatible registry repository for release images.
+
+use async_trait::async_trait;
+use cincinnati::plugins::prelude::*;
+use cincinnati::plugins::InternalIO;
+use cincinnati::plugins::InternalPlugin;
+use custom_debug_derive::CustomDebug;
+use failure::{Fallible, ResultExt};
+use prometheus::IntGauge;
+use std::convert::TryInto;
+use std::path::PathBuf;
+
+pub mod registry;
+
```

Comment:
Make this a
```suggestion
pub mod plugin;
```
and move the plugin implementation there.